### ISSUE TITLE
min escape urls after ? in addressbar suggestion

### ISSUE THREAD

**Issue by raj123** [minbrowser/min#1068]
Suppose I opened three urls
1. google.com/index.php?arg=1
2. google.com/index.php?arg=2
3. google.com/index.php?arg=3

then it will show 3 times "google.com/index.php". However it will open correct url on clicking.

**Comment by PalmerAL** [comment]
The problem with showing the query parameters is that a lot of pages have useless ones that take up a lot of space; for example, here's what most Google searches look like with the query parameters shown:

<img width="591" src="https://user-images.githubusercontent.com/10314059/84825691-6e3cb580-afe7-11ea-8450-86856dcd6433.png">

I guess it might be possible to detect what "useful" query parameters are and only show those, but I can't think of a simple way to do that.

**Comment by raj123** [comment]
i dont have any issue to show all urls with different-2 parameters. However we can apply this removal technique with google url only as google will show multiple searching results only.

**Comment by Syndamia** [comment]
I believe the only viable option is to show the first `X` query parameters (*having it as a setting perhaps*).

Having a system that tries to detect useful query paramteres is bound to fail, either by not detecting a "useful" parameter that is really obscure or by not considering a parameter that is important to the user. Also it will be hard to follow changes in parameters of different sites.
